packages/server/tests/acceptance/serverConfig.spec.ts

```diff
@@ -0,0 +1,41 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import { expect } from 'chai';
+import { Utils } from '../helpers/utils';
+
+describe('@server-config Rate Limiters Acceptance Tests', function () {
```

Comment:
Either we can cancel the `server-config` CI and have this test in ratelimiter, or we keep the `server-config` CI and remove this test here.

Reply:
Agreed, see comment above and updated tests title.